src/main/java/timetables_sort_use_case/frameworks_and_drivers/TimetablesSortMenu.java

```diff
@@ -92,27 +95,28 @@ public TimetablesSortMenu(JFrame frame, AllTimetablesScreen allTimetablesScreen)
     }
 
     /**
-     * Find which buttons were pressed and pass them onto controller.
+     * Finds which buttons were pressed and pass them to allTimetablesScreen
      * @param e the event to be processed
      */
     @Override
     public void actionPerformed(ActionEvent e) {
 
-        boolean bool = true;
+        boolean noInputs = true;
         for (JRadioButton timeButton : timeButtons) {
             if (timeButton.isSelected()) {
                 for (JRadioButton breakButton : breakButtons) {
                     if (breakButton.isSelected()) {
-                        bool = false;
-                        this.setVisible(false);
+                        noInputs = false;
+                        frame.getContentPane().removeAll();
                         allTimetablesScreen.timetablesSort(timeButton.getText(), breakButton.getText());
-                        this.frame.setSize(1280, 720);
+                        frame.add(allTimetablesScreen);
+                        frame.setSize(1280, 720);
```

Comment:
When changing frame size, it might be a good idea to run frame.revalidate().

Reply:
i honestly dont understand revalidate enough so ill just take your word for it since revalidate cant break thigns